docs/conf/portland/2022/news/conference-going-virtual.rst

```diff
@@ -0,0 +1,98 @@
+:template: {{year}}/generic.html
+
+.. post:: Jan 26, 2022
+   :tags: {{shortcode}}-{{year}}, virtual
+
+Conference will be only virtual
+===============================
+
+We won't bury the lede on the announcement today:
+
+   The Portland 2022 conference held on May 23-25 will be fully virtual.
+
+We make this decision with the understanding that it will disappoint some of you.
+We also make this decision with the knowledge that it will relieve some of you from anxiety.
+I will explain a bit more below the reasoning for this decision,
+but the high-level reasons are:
+
+* **After talking with our vendors and staff, we determined we couldn't produce the quality of event that our attendees expect, while doing a hybrid event.**
+* Omicron has changed the risk profile associated with travel & large gatherings, and increased the number of unknown risks that we face as organizers.
+
+We are excited about the quality of our previous virtual events,
+and look forward to iterating and improving our event once again this year.
+We are just as excited to be able to gather safely as you are,
+but we feel the need to hold off for one more year.
+
+We're also looking at the ability for some of our meetups to host watch parties.
+The details are still being worked out,
+but we'd love to find a way for small groups of people to gather in this fashion.
+
+Updated conference details
+--------------------------
+
+Our 2022 conference will be structured the same to our previous virtual events:
+
+* Sunday, May 21: Writing Day
+* Monday, May 22: Conference talks, Lightning Talks, Unconference, Social event
+* Tuesday, May 23: Conference talks, Job Fair, Lightning Talks, Unconference
+
+We're excited to experiment and improve on our existing format as well.
+One change we're considering is doing scheduled Q&A blocks with 2 speakers,
+so attendees don't feel rushed going from the main stage to the Q&A from a talk.
+
+Reasons for going virtual
+-------------------------
+
+I'd like to focus on four major areas where our decision was based:
+
+* Physical Health
+* Event success
+* Mental Health
+* Organizational Health
+
+The biggest worry that we have when organizing the conference is a COVID outbreak.
+We were already planning a number of mitigations,
+but this risk is always present whenever we gather in a large group.
+Omicron has lead to the highest case counts in the USA,
+and increased our anxiety about possible negative outcomes.
```

Comment:
```suggestion
and increased our stress about possible negative outcomes.
```

Reply:
Not sure why I feel that stress works better than anxiety? feel free to ignore if you don't agree